Capybot:
  wheelRadius: 0.048
  wheelDistance: 0.3
  dPoint: 0.1
  mass: 5.0
